src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLLegImpl.java

```diff
@@ -218,4 +222,43 @@ public DataFetcher<Boolean> walkingBike() {
   private Leg getSource(DataFetchingEnvironment environment) {
     return environment.getSource();
   }
+
+  @Override
+  public DataFetcher<Iterable<Leg>> nextLegs() {
+    return environment -> {
+      if (environment.getSource() instanceof ScheduledTransitLeg originalLeg) {
+        int numberOfLegs = environment.getArgument("numberOfLegs");
+        List<String> originModesWithParentStation = environment.getArgumentOrDefault(
+          "originModesWithParentStation",
+          List.of()
+        );
+        List<String> destinationModesWithParentStation = environment.getArgumentOrDefault(
+          "destinationModesWithParentStation",
+          List.of()
+        );
```

Comment:
Use `LegacyGraphQLLegNextLegsArgs` for these

Reply:
This should be implemented now